app/uk/gov/hmrc/tariffclassificationfrontend/models/Decision.scala

```diff
@@ -29,5 +31,6 @@ case class Decision
   methodExclusion: Option[String] = None,
   methodCommercialDenomination: Option[String] = None,
   appeal: Option[Appeal] = None,
-  review: Option[Review] = None
+  review: Option[Review] = None,
+  cancelReason: Option[CancelReason] = None
```

Comment:
Just a thought - I wonder if we should have done a `Cancellation` object which includes a `CancelationReason` and an `applicationForExtendedUse` flag